client/src/components/select/index.js

```diff
@@ -17,13 +18,24 @@ class Select extends Component {
     return (
       <div className='container-select'>
         <label className='label-style'>{labelText}</label>
-        <select className='select-style' onChange={this.selectValue} >
-          <option>{firstItem}</option>
-          {cities.map(city => <option>{city}</option>)}
+        <select className='select-style' onChange={this.selectValue}>
+          <option value='' disabled selected>
+            {firstItem}
+          </option>
+          {cities.map(city => (
+            <option>{city}</option>
```

Comment:
To finish wiring this up: 
Set each option's `value` to {city} also, and then you could also let consuming components pass a selected value/city into this component, like this.props.selectedValue - then in the map call, if city === this.selectedValue, add the `selected` prop to the option element.